src/main/java/mate/academy/spring/service/mapper/UserMapper.java

```diff
@@ -11,6 +13,10 @@ public UserResponseDto mapToDto(User user) {
         UserResponseDto responseDto = new UserResponseDto();
         responseDto.setId(user.getId());
         responseDto.setEmail(user.getEmail());
+        responseDto.setRoleIds(user.getRoles()
+                .stream()
+                .map(Role::getId)
+                .collect(Collectors.toSet()));
```

Comment:
I don't think we should return user roles in response, it could be private information. IDK, you can leave it, and lets see what Kostia think about it